utils/Training.java

```diff
@@ -33,6 +33,18 @@ public static void sgd(NDList params, float lr, int batchSize) {
         }
     }
 
+    /** Allow to do gradient calculations on subManager **/
```

Comment:
can you add a comment why you want to pass in subManager here?

Reply:
I am using that to pass in the childManager that is being used on each epoch and after epoch/calculations are done, the manager is closed so I don't use/add any extra memory.

Reply:
I know that. I mean add above reason to line 36 so ppl who don't know it can easily get the idea

Reply:
Ahhh I thought you were asking why I did it. I'll add it.